RELEASE-NOTES.txt

```diff
@@ -20,6 +20,7 @@
 * Stats: fixed issue that could cause incorrect Stats to be displayed when viewing Stats from a widget.
 * Stats Today widgets: large numbers are now abbreviated.
 * Fixed a bug where files imported from other apps were being renamed to a random name.
+* Fixed a bug where the Post might show a different date in lists than in the Post Settings screen.
```

Comment:
I don't think this is necessary. The difference only occurred with the new Post Scheduling, right? Which isn't public yet.

Reply:
@ScoutHarris Ah yes, you're right. 14.0 would be the first release with the feature enabled.